draft-ietf-tls-tls13.md

```diff
@@ -4879,6 +4879,28 @@ here         No 0-RTT |                 | 0-RTT
                            CONNECTED
 ~~~~
 
+## Post-Handshake
+
+~~~~
+                        CONNECTED---------------+
+                         |   ^                  | Send 
+                         |   |                  | CertificateRequest
+          Send KeyUpdate |   |                  V
+                -- or -- |   |              WAIT_CERT
+          Recv KeyUpdate |   |         Recv |       | Recv Certificate
+        [Send KeyUpdate] |   |        empty |       v
+                -- or -- |   |  Certificate |    WAIT_CV
+   Send NewSessionTicket |   |              |       | Recv 
+                -- or -- |   |              |       | CertificateVerify
+   Recv NewSessionTicket |   |              v       V 
+                -- or -- |   |            WAIT_FINISHED
+ Recv CertificateRequest |   |                  | Recv Finished
+        Send Certificate |   |                  |
+[Send CertificateVerify] |   |                  |
+           Send Finished |   |                  |
+                         +-->+<-----------------+
```

Comment:
A little bit of text here explaining that most of these transitions atomically transition back to CONNECTED would help.  This isn't immediately obvious.